instrumentation/opentelemetry-instrumentation-asgi/MANIFEST.in

```diff
@@ -0,0 +1,9 @@
+graft src
+graft tests
+global-exclude *.pyc
+global-exclude *.pyo
```

Comment:
Not sure what kind of files have these extensions 😅 But looks like we do it everywhere so can't be too much harm to have this?